app/src/main/res/layout/fragment_news.xml

```diff
@@ -1,6 +1,35 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
+
+<androidx.swiperefreshlayout.widget.SwipeRefreshLayout android:id="@+id/vSwipeRefreshLayout"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:orientation="vertical"/>
\ No newline at end of file
+    xmlns:android="http://schemas.android.com/apk/res/android">
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical"
+        android:id="@+id/vFragmentNews">
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/vRecyclerViewNews"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_alignParentStart="true"
```

Comment:
I think this `alignParent...` and `margin...` are doing nothing. If it's the case, let remove it.